src/Rooms/Rooms.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { API } from "aws-amplify";
+import { connect } from 'react-redux';
+import './Rooms.scss';
+import { setPage, setData, saveData, loadData, getUserProfile, submitForm, setFormName } from "../store/form/actions";
+import Loading from "../Loading/Loading";
+import FormPage from '../FormPage/FormPage';
+import { IRoomsProps, IRoomsState } from './types';
+import moment from "moment-timezone";
+
+const mapStateToProps = state => ({
+});
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+});
+
+export class Rooms extends React.Component<IRoomsProps, IRoomsState> {
+  _clockInterval: number;
+  _fetchInterval: number;
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      now: Date.now(),
```

Comment:
@theopolisme here is where time zones would come up - `now` is used to calculate `delta`